variants/t-watch-s3/variant.h

```diff
@@ -0,0 +1,90 @@
+// ST7789 TFT LCD
+#define ST7789_CS 12
+#define ST7789_RS 38  // DC
+#define ST7789_SDA 13 // MOSI
+#define ST7789_SCK 18
+#define ST7789_RESET -1
+#define ST7789_MISO -1
+#define ST7789_BUSY -1
+#define ST7789_BL 45
+#define ST7789_SPI_HOST SPI2_HOST
+//#define ST7789_BACKLIGHT_EN 15
```

Comment:
The #define ST7789_BACKLIGHT_EN 45 is probably needed so that after display.timeout the TFT is turned off